Paper.tex

```diff
@@ -502,10 +502,10 @@ \subsubsection{Block Header Validity}
 \end{equation*}
 \begin{align}
 \expdiffsymb &\equiv \left\lfloor 2^{ \left\lfloor H'_{\mathrm{i}} \div 100000 \right\rfloor - 2 } \right\rfloor \\
-H'_{\mathrm{i}} &\equiv \max(H_{\mathrm{i}} - 3000000, 0)
```

Comment:
maybe worth to introduce the bomb delay variable and mention the value of it in various forks

Reply:
The Yellow Paper "describes one version of the protocol" (see the very end of Section 2). It's easy to make just the bomb delay variable dependant on the fork, but making the entire document fork-aware is a big piece of work. It's not that I'm necessarily against it, but IMO we should either do it later and consistently change the entire document to contain fork switches,  or keep the current approach of describing a single version of the protocol.